e2e/components/autocomplete-e2e.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {browser, by, element} from 'protractor';
+import {screenshot} from '../screenshot';
+import {expectToExist} from '../util/index';
+
+
+describe('autocomplete', () => {
+  const autocompletePanelSelector = '.mat-autocomplete-panel.mat-autocomplete-visible';
```

Comment:
The `.mat-autocomplete-visible` here shouldn't be necessary, the only time an autocomplete is in the DOM and doesn't have that class is when there are no results.